### PR TITLE
Fixed a bug where non-smartstack services would fail to send alerts when they start failing

### PR DESCRIPTION
**Pull request by mjksmith**
Fixed a bug where non-smartstack services would fail to send alerts when they start failing.

### CODE REVIEW

**Comment by Rob-Johnson** [comment]
I know we have this internally, but could you add a comment describing the bug you're fixing please? Also, can you add some tests so we can verify your change works?

File: tests/test_check_marathon_services_replication.py [hunk 1]
```diff
@@ -568,6 +570,26 @@ def test_get_healthy_marathon_instances_for_short_app_id():
...
 
 
+def test_get_healthy_marathon_instances_for_short_app_id_considers_new_tasks_not_healthy_yet():
+    fake_client = mock.Mock()
+    fakes = []
+    one_minute = timedelta(minutes=1)
+    for i in range(0, 4):
+        fake_task = mock.Mock()
+        fake_task.app_id = '/service.instance.foo%s.bar%s' % (i, i)
+        fake_task.started_at = datetime.now() - one_minute*i
```
Comment (solarkennedy) [inline tests/test_check_marathon_services_replication.py]: I'm kinda surprised flake let you get away with `one_minute*i` without a space?

File: tests/test_check_marathon_services_replication.py [hunk 2]
```diff
@@ -568,6 +570,26 @@ def test_get_healthy_marathon_instances_for_short_app_id():
...
+        mock_result = mock.Mock()
+        mock_result.alive = True
+        fake_task.health_check_results = [mock_result]
+        fakes.append(fake_task)
+    fake_client.list_tasks.return_value = fakes
+    actual = check_marathon_services_replication.get_healthy_marathon_instances_for_short_app_id(
+        fake_client,
+        'service.instance',
+    )
+    assert actual == 3
```
Comment (solarkennedy) [inline tests/test_check_marathon_services_replication.py]: I think I'm missing something... why is this `3` and not... 0?

**Comment by solarkennedy** [comment]
ship

**Pull request merged**